http/http-client.private.env.json.template

```diff
@@ -0,0 +1,7 @@
+{
+  "sandpit": {
+    "API-KEY": "value",
+    "clientId": "value",
+    "publicKey": "value"
```

Comment:
Do these need to be private?

Reply:
Chatted about this and might follow up with these changes for ease of use, but will leave for now